openwisp_controller/config/static/config/js/widget.js

```diff
@@ -787,18 +789,31 @@ JSONEditor.defaults.editors.multiple.prototype.$each = function (obj, callback)
     }
 };
 
-// Override setValue method to allow using variables for fields with maxLength
+// Override setValue method to allow using variables for fields with maxLength.
+// The code is copied from jdorn/json-editor library but contains customization
+// to remove maxLength attribute from schema of the a that has a variable value.
```

Comment:
` of the a that has a variable value`, what does this mean?

Reply:
That was a typo, fixed it in https://github.com/openwisp/openwisp-controller/pull/410/commits/a34b0c0a39c274864dc4723d0d03e70738890953 😅